brave/brave-core
# Persist publisher info in the panel

Resolves https://github.com/brave/brave-browser/issues/2684

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- go to clifton.io
- open the panel
- make sure that you see clifton's info
- close the panel
- open it again and make sure that there is no summary flash
- switch to new tab and open brave.com
- open the panel
- make sure that you first see summary and then brave info (no clifton data)

## Reviewer Checklist:
